### PR TITLE
Bump data-plane to 2a08c3e98aefca55ec05cd2c9ea0ac7861def1df

### PR DESCRIPTION
**Pull request by cmluciano**
Signed-off-by: Christopher M. Luciano <cmluciano@us.ibm.com>

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ jobs:
   build:
     working_directory: /go/src/github.com/envoyproxy/go-control-plane
     docker:
-      - image: circleci/golang:1.10
+      - image: circleci/golang:1.11
```
Comment (rshriram) [inline .circleci/config.yml]: ?? why

Comment (cmluciano) [reply .circleci/config.yml]: I had 1.11 locally and thought the build might be failing due to some go-imports differences. I can remove this part.

**Comment by cmluciano** [comment]
Anyone have hints on why the lint is hard failing? I'm able to run it locally and tests are passing too.

**Comment by rshriram** [comment]
Nope. But undo go version change.

**Comment by cmluciano** [comment]
@greghanson discovered the problem is goimports side. I updated to the latest goimports and can replicate the hard exit now. The glide file does not lock goimports so it is also the latest.

**Comment by cmluciano** [comment]
Running make check locally with the updated goimports results in a similar error to CI.

```
make check
--> checking code formatting with 'goimports' tool
?    ./pkg/util/struct_test.go
?    ./pkg/test/server.go
?    ./pkg/test/resource/resource.go
?    ./pkg/test/main/main.go
?    ./pkg/server/server_test.go
?    ./pkg/server/server.go
?    ./pkg/server/gateway.go
?    ./pkg/cache/resource.go
?    ./pkg/cache/cache.go
?    ./pkg/cache/simple_test.go
?    ./pkg/cache/resource_test.go
make: *** [format.check] Error 141
```

But running the command manually results in a clean exit

```
goimports -v -local "github.com/envoyproxy/go-control-plane" -l $(find . -type f -name '*.go' -not -path "./vendor/*") && echo $?

2018/12/07 11:34:48.347044 fixImports(filename="./envoy/config/transport_socket/capture/v2alpha/capture.pb.validate.go"), abs="/Users/cmluciano/go/src/github.com/envoyproxy/go-control-plane/envoy/config/transport_socket/capture/v2alpha/capture.pb.validate.go", srcDir="/Users/cmluciano/go/src/github.com/envoyproxy/go-control-plane/envoy/config/transport_socket/capture/v2alpha" ...
...
...
...
2018/12/07 11:36:52.300887 importPathToNameGoPathParse("google.golang.org/grpc/status", srcDir="/Users/cmluciano/go/src/github.com/envoyproxy/go-control-plane/pkg/server") = "status", <nil>
2018/12/07 11:36:52.301107 importPathToNameGoPathParse("github.com/envoyproxy/go-control-plane/envoy/service/discovery/v2", srcDir="/Users/cmluciano/go/src/github.com/envoyproxy/go-control-plane/pkg/server") = "v2", <nil>
./pkg/server/server_test.go
2018/12/07 11:36:52.306598 fixImports(filename="./pkg/log/log.go"), abs="/Users/cmluciano/go/src/github.com/envoyproxy/go-control-plane/pkg/log/log.go", srcDir="/Users/cmluciano/go/src/github.com/envoyproxy/go-control-plane/pkg/log" ...
0
```

**Comment by cmluciano** [comment]
Generated code but now getting another weird error with go-vet not liking a new dependency bump

```
make vet
--> checking code correctness with 'go vet' tool
# github.com/envoyproxy/go-control-plane/vendor/google.golang.org/grpc/binarylog/grpc_binarylog_v1
vendor/google.golang.org/grpc/binarylog/grpc_binarylog_v1/binarylog.pb.go:206:34: undefined: proto.InternalMessageInfo
vendor/google.golang.org/grpc/binarylog/grpc_binarylog_v1/binarylog.pb.go:474:34: undefined: proto.InternalMessageInfo
vendor/google.golang.org/grpc/binarylog/grpc_binarylog_v1/binarylog.pb.go:534:34: undefined: proto.InternalMessageInfo
vendor/google.golang.org/grpc/binarylog/grpc_binarylog_v1/binarylog.pb.go:581:29: undefined: proto.InternalMessageInfo
vendor/google.golang.org/grpc/binarylog/grpc_binarylog_v1/binarylog.pb.go:645:29: undefined: proto.InternalMessageInfo
vendor/google.golang.org/grpc/binarylog/grpc_binarylog_v1/binarylog.pb.go:711:30: undefined: proto.InternalMessageInfo
vendor/google.golang.org/grpc/binarylog/grpc_binarylog_v1/binarylog.pb.go:751:35: undefined: proto.InternalMessageInfo
vendor/google.golang.org/grpc/binarylog/grpc_binarylog_v1/binarylog.pb.go:800:29: undefined: proto.InternalMessageInfo
make: *** [vet] Error 2
```

**Comment by kyessenov** [comment]
What is the plan for this PR? There is another PR just created to bump API import.

**Comment by ggreenway** [comment]
Closing in favor of #136

**Pull request closed without merging**